documentation/docs/steps/neoDeploy.md

```diff
@@ -54,3 +54,54 @@ steps:
       account: <myDeployAccount>
       host: hana.example.org
 ```
+
+## Example for invalidating the cache
+
+* Setting parameter `invalidateCache` to `true`, will clean up the cache of the deployed application to portal site and reflects the new changes. This is **applicable** only when the application is deployed and accessed via Portal service (Fiori Launchpad site).
+
+Setting this parameter to true, needs additional configurations to go through. Firstly, create an oauth credential as shown below:
+
+* **Oauth credential creation**:
```

Comment:
How about making this a sub-title instead of a bold point with sub-points?

Reply:
I do not have any issues, but I am not sure, what exact document guidelines we need to follow for this. I would rather prefer bullets, as this is part of the sub heading already.

Reply:
To each their own. If you like it like this, keep it like this. My idea was going sub-sub by adding another `#` to the heading. I believe in this case three `#` would do the trick.